cmd/influxd/run.go

```diff
@@ -225,11 +225,17 @@ func openServer(path string, u *url.URL, b *messaging.Broker, initializing, conf
 			openServerClient(s, joinURLs)
 		}
 	} else if !configExists {
-		// We are spining up an server that has no config,
+		// We are spining up a server that has no config,
 		// but already has an initialized data directory
 		joinURLs = []*url.URL{b.URL()}
 		openServerClient(s, joinURLs)
 	} else {
+		if len(joinURLs) == 0 {
+			//This is the first broker, so it always spins up a client to itself for now
```

Comment:
Let's table the `join` naming thing for now. Perhaps I am missing something about the comment. What do you mean by "This". This what? This node? This process? The only way a process can be both a broker and a server is if it's running in single-node mode. Is that what you're saying?